cmd/libsnap-confine-private/classic.c

```diff
@@ -56,3 +57,35 @@ sc_distro sc_classify_distro(void)
 		return SC_DISTRO_CLASSIC;
 	}
 }
+
+bool sc_is_debian_like(void)
+{
+	FILE *f SC_CLEANUP(sc_cleanup_file) = fopen(os_release, "r");
+	if (f == NULL) {
+		return false;
+	}
+	const char *id_keys_to_try[] = {
+		"ID",		/* actual debian only sets ID */
+		"ID_LIKE",	/* distros based on debian */
+	};
+	size_t id_keys_to_try_len =
+	    sizeof id_keys_to_try / sizeof *id_keys_to_try;
```

Comment:
Wouldn't it be more natural to `/ sizeof *char`  here? The result is the same of course but if this wasn't points but say shorts it would not be. Or am I missing something (maybe a bit out-of-touch with C) ?

Reply:
trying to match the style of other array sizeof things we have in nvidia-support.c

Reply:
Indeed, nevermind, my C is just rusty